microsoft-edge/devtools-guide-chromium/whats-new/2023/01/devtools-109.md

```diff
@@ -0,0 +1,201 @@
+---
+title: What's new in DevTools (Microsoft Edge 109)
+description: "Export DevTools instances with enhanced traces.  Debug Recalculate Styles events in Performance tool with selector stats.  Track GC'd objects when Allocation sampling in Memory tool.  Heap Snapshot Visualizer.  Quick View in Focus Mode has all tools.  Navigate directly to Styles in Elements tool.  And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 01/12/2023
+---
+# What's New in DevTools (Microsoft Edge 109)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Import and export your DevTools instances with the new enhanced traces feature
+
+<!-- Subtitle: Use enhanced traces to preserve Console messages, DOM snapshots, and the JavaScript running on the page when exporting a heap snapshot or performance trace. -->
+
+In Microsoft Edge 109, you can turn on new settings to export enhanced traces from the **Performance** and **Memory** tools.  These enhanced traces include more information, such as:
+*  Messages logged to the Console.
+*  The JavaScript that was running on the page at the time of recording.
+*  Snapshots of the DOM.
+
+When saving performance profiles, heap snapshots, allocation timelines, or allocation sampling, you can now export a new `.devtools` file format.  When you then import the `.devtools` file, a new instance of DevTools opens, with the state of your **Elements**, **Console**, and **Sources** tools preserved.  These enhanced traces offer a powerful new way to collaborate and share the information in your DevTools.
+
+To try the enhanced traces feature:
+
+1. In DevTools, click the **Settings** (![Settings icon](../../../media/settings-gear-icon-light-theme.png)) button.
+
+1. In the **Persistence** section of the **Preferences** settings page, select the **Export enhanced performance and memory traces** checkbox:
+
+   ![Settings > Preferences > Persistence checkboxes](./devtools-109-images/enhanced-trace-settings.png)
+
+1. If you want to preserve Console messages, JavaScript, or DOM snapshots, select the corresponding checkboxes.
+
+1. Click the **Close** (**x**) button in DevTools **Settings**.
+
+1. In the **Performance** tool, take a recording.  Or, in the **Memory** tool, take a heap snapshot.
```

Comment:
```suggestion
1. In the **Performance** tool, take a recording.
```
Let's simplify these directions just to focus on the Performance tool for the announcement. Let's add a link at the end of the announcement to the docs in this PR: https://github.com/MicrosoftDocs/edge-developer/pull/2388 when it lands

Reply:
Removed text.  Draft PR link exists and "todo" is noted